webpack-subresource-integrity-embroider/index.js

```diff
@@ -23,7 +23,12 @@ class SubresourceIntegrityPlugin {
         );
         const scriptElements =
           indexHtml.window.document.querySelectorAll("script");
-        const linkElements = indexHtml.window.document.querySelectorAll("link");
+
+        // link integrity is only valid for relationships including
+        // "stylesheet", "module", "modulepreload"
+        const linkElements = indexHtml.window.document.querySelectorAll(
+          "link:is([rel~=module], [rel~=modulepreload], [rel~=stylesheet])",
```

Comment:
For readability per above:

```suggestion
          'link:is([rel~="module"], [rel~="modulepreload"], [rel~="stylesheet"])',
```